dentapp/src/components/Header.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import { LinkContainer } from 'react-router-bootstrap'
+import styled from 'styled-components'
+
+import Navbar from 'react-bootstrap/Navbar'
+import Button from 'react-bootstrap/Button'
+import Container from 'react-bootstrap/Container'
+
+function createBackButton(params) {
+    return (<svg width="9" height="16" viewBox="0 0 9 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M8.7525 1.90245L7.425 0.574951L0 7.99995L7.425 15.425L8.7525 14.0975L2.655 7.99995L8.7525 1.90245Z" fill={params}/>
+    </svg>)
+}

```

Comment:
Since this function will receive always a color, put color, on the params argument of the function

Reply:
Sure!